src/components/Notification/Notification.tsx

```diff
@@ -4,32 +4,41 @@ import React, { useEffect, useRef } from "react";
 import type { HTMLProps, ReactNode } from "react";
 
 import Button, { ButtonAppearance } from "../Button";
-import type { Props as ButtonProps } from "../Button";
 
-export const notificationTypes = {
+import type { Values } from "types";
+
+export const NotificationAppearance = {
+  BORDERLESS: "borderless",
+  DEFAULT: "default",
+  MODAL: "modal",
+  RAISED: "raised",
+} as const;
+
+export const NotificationSeverity = {
   CAUTION: "caution",
   INFORMATION: "information",
   NEGATIVE: "negative",
   POSITIVE: "positive",
+} as const;
+
+type NotificationAction = {
+  className?: string;
```

Comment:
Is the className needed? It practically allows to do anything you want with the button, and I'm not sure we want that.

Reply:
No I don't think it's needed. I left it in because there have been a few occasions where being able to change the class of an internal element of a component has been handy, but this seems like another case of "only add it in if we need to".

Reply:
Yes, I would think so. I don't think that we want people to mess up with notification buttons. At least for now.